sass/main.scss

```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
+    nav {
+        font-size: 1.5em;
+
+        #login {
+	    padding: 15px 30px;
+	}
+
+	#logo {
+	    width: 120px;
+	    height: 120px;
+	}
+
+	#mlh-badge{
+	    width: 140px;
+	}
+
+    }
+
+    #hero {
+        #desk {
```

Comment:
The SVG changes don't seem to have an effect anymore, probably due to our numerous refactors of how scaling works... imo it looks like a good size, the text tweak on `#bh7` is much better for the page "weight". Maybe just remove these SVG adjustments?

Reply:
Although, there is still that shelf clip bug -- could you investigate that too?
(Note that a common bug we saw was that the shapes would get clipped by some sort of invisible background of the shelf, but I _think_ that's not due to the shelf itself. Something to watch out for here. Ain't CSS fun?!)
![Screen Shot 2020-12-20 at 1 42 57 AM](https://user-images.githubusercontent.com/2782749/102706978-c2f60d00-4264-11eb-9547-ff760a8b1cce.png)

Reply:
Note: shelf clipping is a separate bug: #1070

Reply:
[fixed here](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)